public/manifest.json

```diff
@@ -4,18 +4,18 @@
   "icons": [
     {
       "src": "favicon.ico",
```

Comment:
Should the src be updated too?

Reply:
I will check this out

Reply:
This is actually not necessary, so I removed it.